app/backend/approaches/chatreadretrieveread.py

```diff
@@ -121,7 +121,7 @@ async def run_until_final_call(
             model=self.chatgpt_model,
             messages=messages,
             temperature=0.0,
-            max_tokens=32,
+            max_tokens=self.chatgpt_token_limit,
```

Comment:
I don't know if we should up it as far as the limit, since this is just generating a keyword query, but maybe it doesn't hurt to up it to the max. It shouldn't reply with something very long given what it's being prompted for. @srbalakr what do you think?

Reply:
Got your point and modified the limit to 1024.

Reply:
@KesavarajaRK  we need point queries from users conversations. Thats why restricted to smaller length such as 32. Any reason to increase it ?